griptape/tasks/boolean_control_flow_task.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Literal, Union
+from attrs import field
+
+from griptape.artifacts import BooleanArtifact
+from griptape.tasks import BaseControlFlowTask
+
+if TYPE_CHECKING:
+    from griptape.tasks import BaseTask
+
+
+class BooleanControlFlowTask(BaseControlFlowTask):
```

Comment:
I'm not sure we need to reinvent boolean logic in its own Task. Users can implement this themselves in their `control_flow_fn`